src/libraries/System.Drawing.Common/ref/System.Drawing.Common.cs

```diff
@@ -2105,6 +2105,9 @@ public sealed partial class Encoder
         public static readonly System.Drawing.Imaging.Encoder ScanMethod;
         public static readonly System.Drawing.Imaging.Encoder Transformation;
         public static readonly System.Drawing.Imaging.Encoder Version;
+        public static readonly System.Drawing.Imaging.Encoder ColorSpace;
+        public static readonly System.Drawing.Imaging.Encoder ImageItems;
+        public static readonly System.Drawing.Imaging.Encoder SaveAsCmyk;
```

Comment:
@safern, @carlossanlop, I assume we want to add XML comments to these in order to populate the docs?  For consistency maybe we should do so for all the fields here.